createrepo_c.spec

```diff
@@ -11,12 +11,12 @@
 
 %if 0%{?rhel} && 0%{?rhel} <= 7
 %bcond_with python3
-%bcond_with drpm
 %else
 %bcond_without python3
-%bcond_without drpm
 %endif
 
+%bcond_with drpm
```

Comment:
I don't think we want to turn drpm off on Fedora.
This is probably ok on RHEL as long as we want to maintain a downstream patch on RHEL 8 (drpm is enabled there).

Maybe the right thing to do is extending the condition on line 12:
`%if 0%{?rhel} && ( 0%{?rhel} <= 7 || 0%{?rhel} >= 9 )`

Reply:
Ok, I have added the condition. :+1: